invenio_rdm_records/records/systemfields/access/embargo.py

```diff
@@ -39,8 +39,8 @@ def active(self, value):
         """Set the Embargo's (boolean) active state."""
         self._active = bool(value)
 
-    def lift(self):
-        """Update the embargo active status if it has expired.
+    def _lift(self):
```

Comment:
question: this function is called with the field but from another file/module (i.e. line 97 of record.py). Doesn't that break a bit the idea of "private" (underscored function name)?

Reply:
I make it private to make it clear that you should not be calling this function as part of the API: - e.g. ``record.access.embargo._lift()``. The call from the ``RecordAccess`` and ``Embargo`` I consider as intimately related objects (RecordAccess is in charge of creating and working with the Embargo object), and hence it's more about making the API clear for users of the Record class here.

Reply:
Didn't work in the end, because the record.access != draft.access